_posts/en/2022-02-03-project-to-product.md

```diff
@@ -0,0 +1,250 @@
+Enabling teams are temporary in nature.
+The example above is the Accessibility Office.
+
+**Platform teams** provide compelling internal products and services to accelerate delivery by stream-aligned teams.
+One example for Job Bank is that it uses a DocUpload service that allows clients to upload documents, which then performs anti-virus scans and stores them securely.
+Re-using such service enables the organizations to save time and money.
+
+We can then see how Loose-Coupling Architecture (i.e. API-driven) adds great value as it _grants_ more autonomy to teams, reducing the need to projectize by providing on-demand services.
+
+## Final Thoughts
+
+Obviously, we have not covered everything that product management entails, such as vision setting, user research, and the different phases a product goes through in its life cycle.
+This is well documented, such as on [PMI.org](https://www.pmi.org/disciplined-agile/process/product-management) or [PDMA.org](https://community.pdma.org/knowledgehub/home).
+
+The Job Bank team is mature in applying product management philosophy, for example by performing [release impact analysis](https://014gc.sharepoint.com/sites/JobBankDataCentre/SitePages/In-depth-Analysis.aspx?xsdata=MDN8MDF8fDg3YjQ0YWYzM2IzMTQ1YmVhOTMyYzI1ZmNkMzUwNTlifDllZDU1ODQ2OGE4MTQyNDZhY2Q4YjFhMDFhYmZjMGQxfDF8MHw2Mzc3ODQwMTI1MjQyMjcxNTF8R29vZHxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaVYybHVNeklpTENKQlRpSTZJazkwYUdWeUlpd2lWMVFpT2pFeGZRPT0%3D&sdata=OXNZdkVZZlFDTExRYlpRKzBPeTh5UWl5bzlyNGRvRVdwdmZnU3NaaW16WT0%3D&ovuser=9ed55846-8a81-4246-acd8-b1a01abfc0d1%2Cremy.bernard%40hrsdc-rhdcc.gc.ca&OR=Teams-HL&CT=1643030263461&sourceId=&params=%7B%22AppName%22%3A%22Teams-Desktop%22%2C%22AppVersion%22%3A%2227%2F21110108720%22%7D) which evaluates, using empirical evidence, whether a particular release got them closer to a particular goal and if course corrections are needed.
+
+Job Bank is also not just a single product, but a family of products.
+We could increase the level of granularity by breaking each product down to sub-products and to an atomic level: a software library.
+But then we’d get into an argument about external vs internal products…
+
+What this blog sought to do is share how we're striving to create the necessary space for people to interact as one team, even if they may be from different branches, and avoid unnecessary hurdles for much needed collaboration to take place.
+
+We are formalizing how such a transition can take place via a Pilot Framework (consisting of a directive and an associated standard).
+The framework is currently in peer review internally as we want to use it for two Pilot projects: one with Job Bank and one with the [MyESDC mobile app](https://play.google.com/store/apps/details?id=com.esdc_emg&hl=en_US&gl=US).
+
+A draft version of the framework [can be found here]({{ site.baseurl }}/project-to-product.html#stage-1---the-pilot-framework).

```

Comment:
Writing for the web guidelines discourage linking from text like 'here' or 'can be found here'. The words to hyperlink are the words that describe the document that can be found via the link. I'm wondering if would be good to remove line 245 (this sentence) and just hyperlink the words 'Pilot Framework' in the paragraph above? Then each of the framework docs (directive etc.) should probably have a 'draft' banner at the top?  Also for clarity, it probably would be good to use the words 'Pilot Product Management Framework' as the hyperlink.